### PR TITLE
update @walletconnect/ethereum-provider wagmi V1

### PR DESCRIPTION
**Pull request by svenvoskamp**
## Description

update @walletconnect/ethereum-provider

## Additional Information

Before submitting this issue, please make sure you do the following.

- [x] Read the [contributing guide](https://wagmi.sh/dev/contributing)
- [x] Added documentation related to the changes made.
- [x] Added or updated tests (and snapshots) related to the changes made.

### CODE REVIEW

**Comment by SashaDesigN** [comment]
Is this going to be approved in near future?

File: .changeset/stupid-numbers-guess.md [hunk 1]
```diff
@@ -0,0 +1,5 @@
+---
+'@wagmi/connectors': major
```
Comment (tmm) [inline .changeset/stupid-numbers-guess.md:2]:
```suggestion
'@wagmi/connectors': patch
```

File: .changeset/stupid-numbers-guess.md [hunk 2]
```diff
@@ -0,0 +1,5 @@
...
+---
+
+Update @walletconnect/ethereum-provider
```
Comment (tmm) [inline .changeset/stupid-numbers-guess.md:5]:
```suggestion
Updated @walletconnect/ethereum-provider
```

**Review by tmm — CHANGES_REQUESTED** [review]

**Comment by witalick22** [comment]
zalupa yobana, can you do this faster? Why so long?

**Comment by kolirt** [comment]
When will wagmi-v1 be updated?

**Comment by kolirt** [comment]
@tmm ?

**Comment by kolirt** [comment]
@tmm any updates?

**Comment by kolirt** [comment]
@tmm ?

**Comment by glitch-txs** [comment]
better to upgrade to Wagmi v2, we'll keep upgrading and improving the library (ethereum-provider)

**Comment by kolirt** [comment]
> better to upgrade to Wagmi v2, we'll keep upgrading and improving the library (ethereum-provider)

I can't upgrade to v2. Can u update v1 version?

**Comment by glitch-txs** [comment]
You can try by using overrides or (resolutions in yarn)
https://docs.npmjs.com/cli/v8/configuring-npm/package-json#overrides

**Pull request closed without merging**